.pre-commit-config.yaml

```diff
@@ -147,3 +147,9 @@ repos:
         # Help installation by reducing the set of inspected botocore release.
         # There is *a lot* of them, and hatch might fetch many of them.
         - "botocore>=1.25.10"      # 1.25.10 is the current one available for RHEL9
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.6
+    hooks:
+      - id: codespell
+        additional_dependencies:
+          - tomli
```

Comment:
Technically we don't need this for python >= 3.11. But this would depend on if other developers have setup `pre-commit` with the system's python interpreter (or simply installed pre-commit from Fedora)

Reply:
"Why is it needed at all?" was my first question, and I suppose it deserves a comment - `codespell` can use `pyproject.toml` as its configuration, but needs a TOML parser.

Reply:
I thought I mentioned it here, but I guess I've only done it in the [other PR](https://github.com/teemtee/tmt/pull/2746). If we change the GH action to use `setup-python` with `3.x` than this issue also goes away.

Reply:
Good point, I was assuming everyone is using python3.11+ as default and using `hatch` to check lower python versions. On my side, I always use `pre-commit` installed from the system instead of the venv so that it doesn't break the script in `.git/hook/pre-commit` whenever there is a distro update

Reply:
I mix it, sometimes I have `dev` env enabled, sometimes I don't, it just works.

So, let's add a short comment it's there to make older Python happy, and we can resolve this thread?

Reply:
```suggestion
        # Required for python < 3.11
        additional_dependencies:
          - tomli
```